src/components/JazzIcon/JazzIcon.tsx

```diff
@@ -0,0 +1,71 @@
+// Hacks based on
+// https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/components/ui/jazzicon/jazzicon.component.js
+import { createRef, CSSProperties, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import md5 from 'md5';
+
+// @ts-ignore
+import jazzicon from '@metamask/jazzicon';
+
+/**
+ * Wrapper around the jazzicon library to return a React component, as the library returns an
+ * HTMLDivElement which needs to be appended.
+ */
+
+export type Props = {
+  diameter: number | null | undefined;
+  className?: string;
+  style?: Partial<CSSProperties>;
+  address?: string;
+};
+
+export default class Jazzicon extends PureComponent<Props> {
```

Comment:
is there a reason this is a class component?

Reply:
Updated Jazzicon to a functional component